### ISSUE TITLE
You have to switch to a specific branch to 'stop working on issue'

### ISSUE THREAD

**Issue by boltex** [microsoft/vscode-pull-request-github#1901]
This is meant to be a bug report and positive criticism about the ui.

If you want to temporarily switch from working on an issue to work on another, you have to switch first to the right branch before being able to press the icon 'Stop working on issue'. Even so the icon is not disabled., It just does nothing otherwise, no popup message, no messages in the output panel either. Feels like a bug.,

**Comment by alexr00** [comment]
@boltex can you share a gif or some screen shots of the behavior you're seeing?

**Comment by boltex** [comment]
Same png as for #1902, 

When trying to use the 'stop working on issue button on the right of issues listing, I noticed it does nothing if the branch in the lower left is not the matching branch. A notification would be appropriate, also the svg icons should have diferent shades to signal 'disabled' states. 

It would be nice if hover popovers indication could give info about what will (or not) happen if clicked, as a user may be inclined to expect that If ever a button is clickable and enabled, it should give info on hover, and make something visible happen / or notify its been done or not, if invisible.

(again sorry if this is all planned features and I'm being wasteful of your time)

![image](https://user-images.githubusercontent.com/847716/84830178-fd9d9500-aff6-11ea-8174-647e7c9e611d.png)

**Comment by alexr00** [comment]
@boltex I haven't been able to repro this. Do you still see this behavior?

**Comment by boltex** [comment]
Sorry to have not gotten back to you sooner on this - life is a bit hectic in those weird times we're in. 

Anyways, I've had the chance to re-try the same actions in similar conditions (same project, same machine, and other details pretty much the same ) And I'm happy to report to you the behavior seems to be fixed now! 

What I did: I was working on some 'issue xyz' branch, and finalized some stuff, but before doing anything else, I remembered you asking me if this was still an issue: so I've switched to some other branch (something like 'dev' if I remember correctly) and tried the 'Stop working on this issue button' for the 'issue xyz' . The little spinner ran for 2-3 seconds, I was switched to the master branch and the issue no longer had the "working on this" checkmark! Success! 

Thank you very much!  
--
Félix
